table/tables/index.go

```diff
@@ -198,8 +198,12 @@ func (c *index) GenIndexKey(sc *stmtctx.StatementContext, indexedValues []types.
 // Create will return the existing entry's handle as the first return value, ErrKeyExists as the second return value.
 func (c *index) Create(ctx sessionctx.Context, rm kv.RetrieverMutator, indexedValues []types.Datum, h int64,
 	opts ...*table.CreateIdxOpt) (int64, error) {
+	opt := &table.CreateIdxOpt{}
+	if len(opts) != 0 {
+		opt = opts[0]
```

Comment:
> No need to modify here again

Really???

Reply:
You could try. The length of opts could be 0 or 1 only.

Reply:
> When more options is necessary in the future, we have to modify here again, right?